quadlet/syncthing/syncthing.container

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=syncthing-podman.service
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Restart=on-failure
+TimeoutStartSec=900
+
+[Container]
+Image=docker.io/syncthing/syncthing:latest
+
+# Auto-update
+AutoUpdate=registry
+PublishPort=127.0.0.1:8384:8384
```

Comment:
Podman discards port mappings configuration when using `--network=host`:
```
Port mappings have been discarded as one of the Host, Container, Pod, and None network modes are in use
```
So, either remove the `Network` key or the `PublishPort` ones

Reply:
Wow I totally missed this as the service was running fine this whole time. Have commented out the published ports in case people want to use them (I prefer setting Network=Host as Syncthing is run on my local network). Have also added a section in the README to clear this issue